meta-xls.md

```diff
@@ -0,0 +1,413 @@
+*Headers requiring dates will always do so in the format of ISO 8601 (yyyy-mm-dd).*
+
+## 4.4. Format: Amendment XLS
+
+An amendment usually introduces a set of semantically coherent ledger objects and transactions. The specification must make clear the purpose, introduced ledger entries and transactions. Optionally, the amendment specification can introduce new APIs, and include an optional F.A.Q.
+
+*Note: all fields specified above in the “Drafts and Onward” section must also be included here.*
+
+### 4.4.1. Index
+
+A Markdown index to make navigating a long specification easier. This field is optional and left to the author’s discretion.
+
+### 4.4.2. Introduction
+
+A section to succinctly introduce the protocol that the amendment introduces.
+
+* **Terminology** (optional): A section to provide a glossary of terms used across the specification.
+* **System Diagram** (optional): For complex protocols a system diagram that describes the relationship between different components of the system.
+* **Summary**
+  * **Ledger Objects**: A list of ledger objects and their purpose introduced by the amendment.
+  * **Transactions**: A list of transactions and their purpose introduced by the amendment.
+  * **API**: A list of RPCs and their purpose introduced by the amendment (if applicable).
+
+### 4.4.3. Specification
+
+The technical specification describing the syntax and semantics of a new feature. The specification should be detailed enough to allow competing, interoperable implementations. See below for details.
+
+This will be codified in a template later.
+
+#### 4.4.3.1. Ledger Entries
+
+Ledger Entries documents one or more new ledger entry objects introduced or modified by the specification. Each ledger entry must be in its own numbered section. The following sections should be in each section:
+
+##### 4.4.3.1.1. Object Identifier
+
+Each object on the XRP Ledger must have a unique object identifier (ID or key). The ID for a given object within the ledger is calculated using "tagged hashing". This involves hashing some object-specific parameters along with a type-specific prefix called the `key space` (a 16-bit value) to ensure that different objects have different IDs, even if they use the same set of parameters The key space is given as a `hex` representation in the specification. This section introduces the key space and the algorithm for how to calculate a unique object identifier. It is critical that the algorithm does not create collisions between two objects.
+
+##### 4.4.3.1.2. Fields
+
+This subsection lists the object fields, indicating whether a field is constant or optional, its JSON type (for reference, actual storage is binary), Internal Type, and Default Value. The field table MUST include all standard ledger entry fields (like `LedgerEntryType`, `Flags`, `PreviousTxnID`, `PreviousTxnLgrSeq`, `OwnerNode`) as well as fields unique to the ledger entry. The `Account` field is typical for objects owned by a single account.
```

Comment:
```suggestion
This subsection describes the ledger entry fields in a tabular format, indicating whether a field is constant or optional, its JSON type (for reference, actual storage is binary), Internal Type, and Default Value. The field table MUST include all standard ledger entry fields (like `LedgerEntryType`, `Flags`, `PreviousTxnID`, `PreviousTxnLgrSeq`, `OwnerNode`) as well as fields unique to the ledger entry. The `Account` field is typical for objects owned by a single account.
```

Reply:
Some objects have `Account`, some have `Owner`. We should standardize at some point going forward.